src/app/utils/field-validation/validators/common.ts

```diff
@@ -12,3 +18,40 @@ export const notEmptySingleAnswerResponse: ResponseValidator<ProcessedSingleAnsw
     )
   return right(response)
 }
+
+type SignatureValidatorConstructor = (
+  formField: IEmailFieldSchema | IMobileFieldSchema,
+) => ResponseValidator<ProcessedSingleAnswerResponse>
+
+/**
+ * A function which returns a signature validator constructor for mobile and email verified field.
+ * The validator checks if field has correct signature
+ */
+export const makeSignatureValidator: SignatureValidatorConstructor = (
```

Comment:
could you inline this function type? no point separating it if it's not reused or exported